src/main/java/mate/academy/internetshop/controllers/ProductListController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.service.ProductService;
+
+public class ProductListController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllProductsController extends HttpServlet {
```

Reply:
let's add verbs to Controller's names for example  GetAllUsersController (here and everywhere)

Reply:
done